admin-ui/plugins/auth-server/components/AuthN/Basic.js

```diff
@@ -30,23 +28,12 @@ function AuthNListPage({ isBuiltIn = false }) {
   const themeColors = getThemeColor(selectedTheme)
   const bgThemeColor = { background: themeColors.background }
 
-  const [list, setList] = useState({
-    ldap: [],
-    scripts: [],
-  })
-
   const authN = useSelector((state) => state.authNReducer.acrs)
-  const ldap = useSelector((state) => state.ldapReducer.ldap)
-  const scripts = useSelector((state) => state.customScriptReducer.items)
-  const scriptsLoading = useSelector((state) => state.customScriptReducer.loading)
-  const loading = useSelector((state) => state.ldapReducer.loading)
   const acrs = useSelector((state) => state.acrReducer.acrReponse)
-  const customScriptloading = useSelector((state) => state.customScriptReducer.loading)
   const { permissions: cedarPermissions } = useSelector((state) => state.cedarPermissions)
 
-  SetTitle(t('titles.authn'))
+  SetTitle('Built In')
```

Comment:
Hard-coding a string for the page title is bad because it violates internationalization best practices by embedding a display string directly in the source code, which bypasses the localization framework and prevents the title from being translated for different locales.